File: kotlin/app/src/main/kotlin/org/example/models/AccelSample.kt
package org.example.models

/**
 * Модель одного измерения акселерометра.
 *
 * @property x ускорение вдоль длинной стороны устройства
 * @property y ускорение вдоль короткой стороны устройства
 * @property z ускорение перпендикулярно экрану
 * @property timeStamp время измерения данных
 */
data class AccelSample(
  val x: Double,
  val y: Double,
  val z: Double,
  val timeStamp: Long
)

File: dart/bin/models/accel_sample.dart
/// Измерение акселерометра
class AccelSample {
  /// Ось вдоль длинной стороны устройства
  final double x;

  /// Ось вдоль короткой стороны
  final double y;

  /// Ось перпендикулярно плоскости экрана
  final double z;

  /// Время измерения данных
  final int timeStamp;

  const AccelSample({
    required this.x,
    required this.y,
    required this.z,
    required this.timeStamp,
  });
}
